### PR TITLE
adding the spectrum analyzer message

### PR DESCRIPTION
**Pull request by mbavaro**
self explanatory, not decorated much yet

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/specan.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Packages define a logical collection of SBP messages. Packages can
```
Comment (mookerji) [inline spec/yaml/swiftnav/sbp/specan.yaml]: @mbavaro looks like you accidentally copied these comments over from the tutorial example....

File: python/sbp/specan.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+
+"""
```
Comment (mookerji) [inline python/sbp/specan.py]: Note, you're going to have to manually update: https://github.com/swift-nav/libsbp/blob/master/python/sbp/table.py#L17, which is not programmatically generated right now.

**Comment by mfine** [comment]
What is `specan`? Is that a term of art around spectrum analysis?

**Comment by mbavaro** [comment]
@mfine yeah, not the best choice but it's the same as in PFWP
@mookerji I rebased, modified table.py and related examples .. do you think it is OK now?

**Comment by mfine** [comment]
@mbavaro are there going to be more spectrum analyzer messages? It seems overkill to start a new file/group for this - can we just put it in `piksi` instead?

**Comment by mbavaro** [comment]
@mfine more messages might be defined in the future: we don't deal much with interference at all, but I could envisage some messages to configure the spectrum analyzer traces (channels, bandwidths, averages, windows, ...), as well as messages for changing IF filter parameters or setting notches there. Unless it hurts anyone's sense of beautiful code, I would leave like this.

**Comment by mfine** [comment]
@mbavaro well we can always add the grouping later - I'd would roll this into piksi.yaml. Other "single" groupings like ext-events haven't really been worth the split out and file pollution.

**Comment by mookerji** [comment]
@mbavaro ack, yeah, I was going to echo @mfine's comment before but didn't finish the review. As an alternative to `piksi.yaml`, I would also recommend `acquisition.yaml` or `logging.yaml`. If the configuration you're describing comes to fruition, then the acquisition module might be more appropriate.

**Comment by mbavaro** [comment]
I see it hurts your sense of beauty. OK I am moving it to piksi.yaml.

**Comment by mfine** [comment]
Looks great, thanks!

**Comment by mookerji** [comment]
:shipit:

**Pull request merged**